src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt

```diff
@@ -16,7 +19,7 @@ private val myLogger = LogManager.getLogger("biokotlin.util.MergeGVCFUtils")
 
```

Comment:
This file should have a header comment with a description of the scope of functionality that is included.